### PR TITLE
renamed package.json to lint

### PR DESCRIPTION
**Pull request by akanellop**
installed prettier-check

### CODE REVIEW

**Comment by akanellop** [comment]
If i rename the package.json file,I cannot run "yarn format", and I'm guessing neither "yarn test".
Is this how is supposed to be?

File: lint [hunk 1]
```diff
@@ -8,10 +8,14 @@
   "license": "MIT",
   "devDependencies": {
     "jest": "^23.4.1",
-    "prettier": "1.13.7"
+    "prettier": "^1.13.7",
+    "prettier-check": "^2.0.0"
   },
   "scripts": {
     "test": "jest",
+    "test2": "prettier-check src/*.js",
```
Comment (parisk) [inline lint]: Can you rename this to `lint` please?

I did not express that well in https://github.com/sourcelair/env-spec-js/issues/9 😅.

File: lint [hunk 2]
```diff
@@ -8,10 +8,14 @@
   "license": "MIT",
   "devDependencies": {
     "jest": "^23.4.1",
-    "prettier": "1.13.7"
+    "prettier": "^1.13.7",
+    "prettier-check": "^2.0.0"
```
Comment (parisk) [inline lint]: Can you add this via `yarn add --dev prettier-check`, so it updates `yarn.lock` as well?

**Review by parisk — CHANGES_REQUESTED** [review]
My bad! I did not express that well in #9.

On top of my previous comments we will have to:

1. Delete `package-lock.json`
2. Add `yarn lint` as the first script (before `yarn test`) in `travis.yml`. See how we do it in bouncer-api: https://github.com/sourcelair/bouncer-api/blob/master/.travis.yml#L8

## How to remove `package-lock.json`

## How to do this.

1. Remove the `package-json.lock` file from Git:
    ```
    git rm package-lock.json
    ```
2. Commit and Push

File: lint (hunk 1 above)
Comment (akanellop) [reply lint]: the file's name will be lint or package.json?

Comment (parisk) [reply lint]: The file's name should be `package.json`.

Inside the `scripts` section of the `package.json` file we will need the following entry:

```json
    "lint": "prettier-check src/*.js",
```

**Comment by akanellop** [comment]
I think we're ready

**Review by parisk — APPROVED** [review]
Amazing 💯! Thanks 🇯🇲!

**Pull request merged**